Created Blog for Lab Meeting (#73)
WhitakerLab/whitakerlab.github.io: _posts/blog/2019-08-01-lab-meeting.md

> # Whitaker Lab Meeting | 1 August 2019

Reviewer: This file needs to be named `2019-08-01-2019-08-01-lab-meeting.md` because jekyll automatically strips the date from the start of the file name to create the blog post url but we _do_ want the date to be in that url (otherwise all lab meeting blog posts are called `lab-meeting` and only the most recent shows up.

We don't need the heading because it shows up as the title of the post, so just drop this first line too.

You _do_ need the ["front matter"](https://jekyllrb.com/docs/front-matter/) from the template and updated with this week's information. Check out `_posts\blog\template_lab-meeting.md` for the template. You shouldn't have to change too much.

This week’s lab meet was petite but by no means picayune! 
We discussed ways to learn physics, Neurohackademy, and how to build trust and confidence in a group. 
Thanks Kirstie, Isla, and Sarah for your tips, suggestions, and a super interesting conversation!

### Celebrations and cool things to share:

> Kirstie: is finding Neurohackademy great so far – with lots of awesome talks! Here are a select few :point_down:

Reviewer: I think it looks nice to format the names at the start of these bullet points in bold.

Sorry for being so picky. If you can change this for the others that would be great!

```suggestion
**Kirstie**: is finding Neurohackademy great so far – with lots of awesome talks! Here are a select few :point_down:
```

>   * Codeclimate for readable code: https://codeclimate.com

Reviewer: I _think_ these are going to need to be formatted as links (GitHub does it automatically but jekyll won't know to convert them to links.

I've suggested for this first one and then you can do the rest.

```suggestion
  * Codeclimate for readable code: [https://codeclimate.com](https://codeclimate.com)
```

  * Intro to git: http://rogerdudler.github.io/git-guide
  * The most fantastic git resource: https://ohshitgit.com
  * A bunch of thing you can do with Jupyer: https://blog.jupyter.org/i-python-you-r-we-julia-baf064ca1fb6
  * A great project template for a research project: https://drivendata.github.io/cookiecutter-data-science

Georgia (and Kirstie) got ethics approval for the Citizen Science Project – the day has finally arrived! YAY! 

> The GitHub repository and survey are now live, and we’ve sent an email to the project’s mailing list to launch it. :rocket:

Reviewer: Add links to repo and survey

### Questions we’re thinking about: 

Isla: is in the midst of writing her dissertation, and is wondering how to pinpoint the right level of detail needed.

> Because she’s used to sharing knowledge and making things clear :green_heart:, it can be heard to work out the right level     of explanation for other experts. 

Reviewer: I'm not sure if the `:green_heart:` syntax renders in jekyll. I think you have to copy the emoji. I do that from https://emojipedia.org/green-heart/

Also fixe weird whitespace (which probably comes from writing in word and then copying and pasting.)

>   * Tip for writing - start with bullet points and then build up a path of what else needs explaining.

Reviewer: I think this bullet point needs a little more flow from the sentences above.

```suggestion
  * One tip that the group had for writing is to start with bullet points and then build up a path of what else needs explaining.
```

Georgia: wants to know how to help people help each other in the lab meetings. 
  * Sarah suggested that since she doesn’t have the domain knowledge to help everyone, if we talked about technical challenges, she would be better able to help.
  * Georgia suggested having some kind of shared goal which we can all work together on. 
  * Isla suggested being explicit and bringing an extremely practical problem everyone could on.

>   * Sarah  suggested something like a coding club - maybe there could be a small problem but one entirely contained within the meeting? Something like RegTech talks. 

Reviewer 2: ```suggestion
  * Sarah  suggested something like a coding club - maybe there could be a small problem but one entirely contained within the meeting? Something like REG tech talks. 
```

>   * Sarah  suggested something like a coding club - maybe there could be a small problem but one entirely contained within the meeting? Something like REG tech talks. 

Reviewer: Weird whitespace.

Sarah: Wants tips on working on a project if the content is overwhelming (a.k.a anyone familiar with Guassian Processes? :sweat_smile:)

Kirstie has been thinking a lot about the need for trust in communities – specifically, that we need to trust each other to be able to ask each other questions. 
She’s been thinking about what the barriers to trust are in the group are. She’d love to know what has worked in the past to build a supportive team and environment.

>   * Georgia: Overall, it feels like there is a lot of trust at the Turing - people do help each other out.

Reviewer: Can you reformat this into a summary of the discussion rather than individual points?

Fine to cut down on details - it doesn't need to be a transcript.

Happy to iterate if you're not sure 😄

  * Georgia: if shyness and lack of self-confidence is an issue for contributing to a group, presenting, asking questions etc., is exposure therapy productive or counterproductive for overcoming it? 
    * Sarah: you have to strike a balance, e.g. finding a friendly audience to start with (but not avoiding it entirely)

>   * Georgia asked Sarah what makes the Binder community supportive?

Reviewer: Link to binder

        * EMOJIS!!! :sparkles: :sparkling_heart: :smiley:
        * Friendly messages
        * Never miss an opportunity to remind people they're doing a good job - very important in the Binder community because they're so distributed.

Georgia would like ideas about the best ways of learning physics?
  * Sarah suggests looking at science videos on YouTube! There are also fun experiments that you can do yourself at home, for example experimenting with static electricity using balloons - stay away from dry manuals
  * Check out: https://www.youtube.com/results?search_query=kurzgesagt 
	* Find someone who’s a physicist and ask them to explain directly
  * Isla gave this awesome (wise) advice: You start learning about physics when you're a baby when you pick things up and drop them to see what happens – carry on with this attitude. Stay curious! :baby::woman::older_woman:

>   * Georgia noticed that a lot of popular science books explain things in terms of metaphors, but do the metaphors capture everything? (Consensus in the room: probably not?)

Reviewer: I suspect you can drop this unless you want to expand it. As it is it's a little bullet point-y.

### We also had a general discussion about astrophysics, maths, and academia:

> * Turns out there are different ways of thinking about differential geometry depending on if you're a mathematician or a physicist. 

Reviewer: Summarise into a few sentences, not bullet points.
